griptape/tokenizers/cohere_tokenizer.py

```diff
@@ -1,15 +1,18 @@
-import cohere
+from __future__ import annotations
+from typing import TYPE_CHECKING
 from attr import define, field
 from griptape.tokenizers import BaseTokenizer
 
+if TYPE_CHECKING:
+    from cohere import Client
 
 @define(frozen=True)
 class CohereTokenizer(BaseTokenizer):
     DEFAULT_MODEL = "command"
     MAX_TOKENS = 2048
 
     model: str = field(default=DEFAULT_MODEL, kw_only=True)
-    client: cohere.Client = field(kw_only=True)
+    client: Client = field(kw_only=True)
```

Comment:
If this is conditionally imported above, do we need `import_optional_dependency()` here?

Reply:
We're just using the type here, not the actual module, so no need for `import_optional_dependency()`

Reply:
Intersting, thank you! Lots to learn about the Python import model.